rename_deprecated_modules.py

```diff
@@ -32,7 +32,7 @@
     # easy_my_coop_payment_term
     # easy_my_coop_taxshelter_report
     # easy_my_coop_website
-    # easy_my_coop_website_portal
+    "easy_my_coop_website_portal": "cooperator_portal",
```

Comment:
Replacing the line causes merge conflicts. Need to preserve the comment and append this as an extra line.

Reply:
👌